src/components/sidebar/components/sidebar-back-to-link/index.tsx

```diff
@@ -10,14 +12,19 @@ const BackToLinkText: React.FC = ({ children }) => (
   </Text>
 )
 
-// TODO: Design is still planning this functionality & states.
-// TODO: Since not all cases are covered, we return a placeholder
-// TODO: link if necessary props aren't provided.
-const SidebarBackToLink: React.FC<{
-  text?: string
-  url?: string
-}> = ({ text, url }) => {
-  if (!text || !url) return <PlaceholderBackToLink />
+/**
+ * TODO: Design is still planning this functionality & states. Since not all
+ * cases are covered, we return a placeholder link if necessary props aren't
+ * provided.
+ */
+const SidebarBackToLink = ({
+  text,
+  url,
+}: SidebarBackToLinkProps): ReactElement => {
```

Comment:
This update of the addition of `SidebarBackToLinkProps` brings the component up to speed with how we've been building components lately: adding a `types.ts` file next to the component that contains all the types for the component, including the props interface for the component.